master_server/master_server.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/python
+
+"""This script works on the asssumption of a certain directory structure.
+
+
+     It then acts the master server and load balances the incoming
+     requests.
+     A database is created to store the incoming requests from user.
+"""
+import os
+import requests
+from flask import Flask
+from flask import request
+from flask import render_template
+from flask_sqlalchemy import SQLAlchemy
+import Request_pb2
+
+class MyFlaskApp(Flask):
+  """
+    This class configures the flask app and executes function
+    pre_task before starting flask server.
+  """
+  def run(self, host=None, port=None, debug=None, load_dotenv=True, **options):
+    if not self.debug or os.getenv('WERKZEUG_RUN_MAIN') == 'true':
+      with self.app_context():
+        pre_tasks()
+    super(MyFlaskApp, self).run(
+        host=host, port=port, debug=debug, load_dotenv=load_dotenv, **options)
+
+app = MyFlaskApp(__name__)
+
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///database.db'
+db = SQLAlchemy(app)
+
+
+working_vm_address = []
```

Comment:
let's name them working_vm_address_list and available_vm_address_list

Reply:
Changed name of list as suggested.